docs/includes/eloquent-models/AuthenticatableUser.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Models;
+
+use MongoDB\Laravel\Auth\User;
```

Comment:
This use statement must be aliased to not conflict with the class name. The alias is correctly used after `extends`.
```suggestion
use MongoDB\Laravel\Auth\User as Authenticatable;
```